app/ui/page.tsx

```diff
@@ -317,7 +325,49 @@ export default function UI() {
             </Button>
           </div>
         </div>
-        <div className="space-y-4 pb-5 border-b">
+        <div className="space-y-4 pt-12">
+          <div className="text-2xl font-semibold">ToogleGroup</div>
+          <p>
+            Based on Radio Group component from Headless ui,{" "}
+            <a
+              className="text-surface-primary-main hover:underline"
+              href="https://headlessui.com/react/radio-group"
+            >
+              check their docs
+            </a>
+            .
+          </p>
+          <div className="divide-x divide-surface-surface-2 flex items-center space-x-5">
+            {["xs", "sm", "md", "lg"].map(size => (
+              <div key={size} className="pl-4">
+                <p>Size: {size}</p>
+                <ToggleGroup value={slipage} onChange={setSlipage}>
+                  <ToogleGroupOption
+                    size={size as ToogleOptionSizeProp}
+                    value="auto"
+                  >
+                    auto
+                  </ToogleGroupOption>
+                  <ToogleGroupOption
+                    size={size as ToogleOptionSizeProp}
+                    value="0.1"
+                  >
+                    0.1%
+                  </ToogleGroupOption>
+                  <ToogleGroupOption
+                    size={size as ToogleOptionSizeProp}
```

Comment:
If we are casting types here, there is something wrong in our types.

Reply:
if we add  types to `["xs", "sm", "md", "lg"]` we don't need to cast. So we just declare the type in 1 place